x/concentrated-liquidity/incentives_test.go

```diff
@@ -163,6 +192,21 @@ func addToUptimeAccums(ctx sdk.Context, poolId uint64, clKeeper *cl.Keeper, addV
 	return nil
 }
 
+// addDecCoinsArray adds the contents of the second param from the first (decCoinsArrayA + decCoinsArrayB)
+// Note that this takes in two _arrays_ of DecCoins, meaning that each term itself is of type DecCoins (i.e. an array of DecCoin).
+func addDecCoinsArray(decCoinsArrayA []sdk.DecCoins, decCoinsArrayB []sdk.DecCoins) ([]sdk.DecCoins, error) {
```

Comment:
Im very surprised this isn't a function that exists natively!

Reply:
Me too!